### PR TITLE
Initial setup of account overview page.

### PR DESCRIPTION
**Pull request by jonearley**
Includes a first pass on the account overview page and related style and template changes.

Screen Shot 2020-11-09 at 13 22 31

### CODE REVIEW

File: css/_base.scss [hunk 1]
```diff
@@ -81,120 +81,22 @@ h4 {
 }
 
 .layout {
```
Comment (jonearley) [inline css/_base.scss:83]: This was copied from design system docs, but simplifying it for this project.

File: css/_base.scss [hunk 2]
```diff
@@ -81,120 +81,22 @@ h4 {
...
-
-  & > * {
-    grid-column: content-start / content-end;
-  }
-}
+  padding: 0 var(--space-medium);
+  max-width: 1280px;
+  margin: 0 auto;
 
-.table-of-contents {
```
Comment (jonearley) [inline css/_base.scss:97]: Don't need any of this.

File: css/_defaults.scss [hunk 3]
```diff
@@ -54,7 +54,12 @@ th,
 td {
   padding: var(--space-medium);
   padding-left: 0;
+  padding-right: 0;
   vertical-align: top;
+
+  &:not(:last-of-type) {
```
Comment (jonearley) [inline css/_defaults.scss:60]: Make sure the last cell in a table row can have content to the edge of the page.

File: my_account.rb [hunk 4]
```diff
@@ -19,12 +19,12 @@
   #loans = Prototypes::Loans.new
   #session[:uniqname] = 'mrio' #need to get this from cosign?
   #patron = Patron.for(uniqname: session[:uniqname])
-  #erb :home, :locals => {loans: loans, patron: patron}
```
Comment (jonearley) [inline my_account.rb:22]: @niquerio Is there a way to grab the active session patron data? I can update the "Wecome, [name]" part, then.

File: views/home.erb [hunk 5]
```diff
@@ -1,3 +1,49 @@
-<h1>Welcome, <%=patron.full_name %></h1>
+<h1>Welcome, Samuel Higgins</h1>
 <p>This is the home of your University of Michigan Library account.</p>
 
+<!--
+  Could this data come from back-end?
```
Comment (jonearley) [inline views/home.erb:5]: @niquerio Nothing super important to do immediately, but I figure long-term group this related navigation together in the backend?

Comment (jonearley) [reply views/home.erb:5]:
```ruby
# Ideas for navigation data
[
  {
    label: "Fines & Fees", # heading
    to: "/fines", # path
    notifications: 2, # used for the badge.
    description: "Views information ...", # the homepage description text.
    featured: true, # to tell if it's on the homepage.
    icon: "money" # unique ID for associated icon
  }
]
```

**Review by niquerio — APPROVED** [review]
It is fine!

**Pull request merged**